Shared/Samples/Project with chosen transformation/ProjectWithChosenTransformationView.swift

```diff
@@ -0,0 +1,147 @@
+                    // Creates the transformation.
+                    let transformation = GeographicTransformation(step: transformationStep)
+                    
+                    // Projects to a coordinate system used in Mongolia, with WKID 28413 using a transformation.
+                    specificProjectedPoint = GeometryEngine.project(originalPoint, into: .mongolia, datumTransformation: transformation)
+                    
+                    // Updates the geometry of the point graphic.
+                    model.pointGraphic.geometry = specificProjectedPoint
+                    
+                    // Updates the location callout placement.
+                    calloutPlacement = CalloutPlacement.location(specificProjectedPoint)
+                } else {
+                    // Hides the callout and point graphic.
+                    calloutPlacement = nil
+                    model.pointGraphic.geometry = nil
+                }
+            }
+            .callout(placement: $calloutPlacement.animation(.default.speed(2))) { _ in
+                VStack(alignment: .leading) {
+                    Text("Coordinates")
+                        .fontWeight(.medium)
+                    Text("__Original (WGS 84):__\n\(originalPoint.xyCoordinates)")
+                    Text("__Projected (non-specific):__\n\(unspecifiedProjectedPoint.xyCoordinates)")
+                    Text("__Projected (WKID: 108055):__\n\(specificProjectedPoint.xyCoordinates)")
+                }
+                .font(.callout)
+                .padding(6)
+            }
+            .overlay(alignment: .top) {
+                Text("Tap on the map.")
+                    .frame(maxWidth: .infinity)
+                    .padding(.vertical, 6)
+                    .background(.thinMaterial, ignoresSafeAreaEdges: .horizontal)
+            }
+    }
+}
+
+private extension ProjectWithChosenTransformationView {
+    /// The model used to store the geo model and other expensive objects
+    /// used in this view.
+    final class Model {
+        /// A map with a WGS84 world basemap and an initial viewpoint over Mongolia.
+        let map: Map = {
+            let portalItem = PortalItem(
+                portal: .arcGISOnline(connection: .anonymous),
+                id: .worldBasemap
+            )
+            let map = Map(item: portalItem)
+            
+            // The transformation used in this sample is only valid in the following area.
+            let extent = Envelope(
+                xRange: 87.76...119.94,
+                yRange: 41.58...52.15,
+                spatialReference: .wgs84
+            )
+            map.initialViewpoint = Viewpoint(boundingGeometry: extent)
+            return map
+        }()
+        
+        /// A graphics overlay containing a graphic with a circular, red marker symbol.
+        let graphicsOverlay = GraphicsOverlay(graphics: [
+            Graphic(symbol: SimpleMarkerSymbol(color: .red, size: 8))
+        ])
```

Comment:
```suggestion
        let graphicsOverlay = GraphicsOverlay()
```

This graphic can be removed now.